### PR TITLE
Sk latex behaviour

### PR DESCRIPTION
**Pull request by JuneKelly**

### CODE REVIEW

File: lib/ace/mode/behaviour/latex.js [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    var line = session.doc.getLine(cursor.row);
+    // Reset previous state if text or context changed too much
+    if (!this.isAutoInsertedClosing(cursor, line, context.autoInsertedLineEnd[0]))
+        context.autoInsertedBrackets = 0;
+    context.autoInsertedRow = cursor.row;
+    context.autoInsertedLineEnd = bracket + line.substr(cursor.column);
+    context.autoInsertedBrackets++;
+};
+
+LatexBehaviour.recordMaybeInsert = function(editor, session, bracket) {
```
Comment (jpallen) [inline lib/ace/mode/behaviour/latex.js]: This seems to be a dead method now.

File: lib/ace/mode/behaviour/latex.js [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+};
+
+LatexBehaviour.isAutoInsertedClosing = function(cursor, line, bracket) {
+    return context.autoInsertedBrackets > 0 &&
+        cursor.row === context.autoInsertedRow &&
+        bracket === context.autoInsertedLineEnd[0] &&
+        line.substr(cursor.column) === context.autoInsertedLineEnd;
+};
+
+LatexBehaviour.isMaybeInsertedClosing = function(cursor, line) {
```
Comment (jpallen) [inline lib/ace/mode/behaviour/latex.js]: Also dead

File: lib/ace/mode/behaviour/latex.js [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+        line.substr(cursor.column) === context.maybeInsertedLineEnd &&
+        line.substr(0, cursor.column) == context.maybeInsertedLineStart;
+};
+
+LatexBehaviour.popAutoInsertedClosing = function() {
+    context.autoInsertedLineEnd = context.autoInsertedLineEnd.substr(1);
+    context.autoInsertedBrackets--;
+};
+
+LatexBehaviour.clearMaybeInsertedClosing = function() {
```
Comment (jpallen) [inline lib/ace/mode/behaviour/latex.js]: Dead

**Review by jpallen — CHANGES_REQUESTED** [review]
LGTM, just a few bits of dead code still 👍

**Comment by JuneKelly** [comment]
@jpallen dead code removed

**Review by jpallen — APPROVED** [review]

**Pull request merged**